Procurement/Procurement.csproj

```diff
@@ -109,6 +109,7 @@
     <Compile Include="Controls\FragmentStashTab.xaml.cs">
       <DependentUpon>FragmentStashTab.xaml</DependentUpon>
     </Compile>
+    <Compile Include="ViewModel\Recipes\ChromaticRecipe.cs" />
```

Comment:
I know the list of files here isn't sorted and is primarily managed by Visual Studio, but maybe manually move this line back to where it was, so we don't make the sorting worse.

Reply:
I let VS handle the project files. I don't mess with them :)